src/controller/java/src/chip/devicecontroller/ChipIdLookup.java

```diff
@@ -0,0 +1,67 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package chip.devicecontroller;
+
+public final class ChipIdLookup {
+  /**
+   * Translates cluster ID to a cluster name in upper camel case. If no matching ID is found,
+   * returns an empty string.
+   */
+  public static String clusterIdToName(long clusterId) {
+    ClusterIDMapping.BaseCluster cluster = ClusterIDMapping.getCluster(clusterId);
+    if (cluster == null) {
+      return "";
```

Comment:
maybe a formatted name of `UNKNOWN_CLUSTER(%d)` or similar would be more human friendly. Here and below, since overall empty strings in logs are not obvious to spot.

Reply:
Modified.